tomopy/misc/npmath.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import numpy  as np
+import logging
+import warnings
+
+logger = logging.getLogger(__name__)
+
+
+__author__ = "Chen Zhang"
+__all__    = [ 'gauss',
+               'calc_cummulative_dist',
```

Comment:
calc_ parts can be removed to shorten the function names.

scipy.stat uses discrete_cdf as a function name for cumulative distrubutions and we can maybe use the same.